src/EditorFeatures/Test/PasteTracking/PasteTrackingTestState.cs

```diff
@@ -0,0 +1,130 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using Microsoft.CodeAnalysis.Editor.Shared.Utilities;
+using Microsoft.CodeAnalysis.Editor.UnitTests.Utilities;
+using Microsoft.CodeAnalysis.Editor.UnitTests.Workspaces;
+using Microsoft.CodeAnalysis.Text;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.Editor;
+using Microsoft.VisualStudio.Text.Editor.Commanding.Commands;
+using Moq;
+using Roslyn.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.PasteTracking
+{
+    internal sealed class PasteTrackingTestState : IDisposable
```

Comment:
Do we need this?  We already have TestState in Microsoft.CodeAnalysis.EditorFeatures.Test.Utilities.  It already has SendPaste.

Do we need this type?  I'd prefer if we just used that one.